### PR TITLE
Styleheadline

### PR DESCRIPTION
**Pull request by lolashot**
Create Css and Html File
Setup Linters
Update Readme

styleheadline branch updated to pass all checks and able to merge with the main branch. thank you.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1,27 +1,62 @@
...
+  background: #fff;
+  mix-blend-mode: multiply;
+}
+
+.headline {
+  color: #111;
+  font-size: 1.5em;
+  font-weight: 500;
+  margin-top: 200px;
 }
```
Comment (Abenezer-Tilahun) [inline style.css:43]: - [ ] Please try to Adjust content to aligned with flexbox.
- [ ] The Headline section must have the margins (top, bottom, left, right) defined in the design , You must stick to the design as much as possible (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)
Your Headline section  look likes this : 
try

But the Figma design Look like this : 

Figma1

- [ ] Kindly, For  headline section you must  uses the font and colors (background and font) defined for the template. You must stick to the design as much as possible (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)

File: style.css [hunk 2]
```diff
@@ -1,27 +1,62 @@
...
+
+.socials {
+  margin-right: 15px;
+  color: #acb7c3;
+}
+
+li {
+  text-decoration: none;
+  list-style-type: none;
+}
```
Comment (Abenezer-Tilahun) [inline style.css:62]: - [ ] Kindly , try to adjust font and space betwwen the social icons ,You must stick to the design as much as possible (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)
Your social icons look likes this  : 

Abc

But the Figma design look likes : 
Socail

File: index.html [hunk 3]
```diff
@@ -10,23 +10,36 @@
...
+                <a href="#"><i class="fa-brands fa-linkedin socials"></i></a>
+            </li>
+            <li>
+                <a href="#"><i class="fa-solid fa-envelope socials"></i></a>
+            </li>
+            <li>
+                <a href="#"><i class="fa-brands fa-github socials"></i></a>
+            </li>
+        </ul>
     </section>
```
Comment (Abenezer-Tilahun) [inline index.html:43]: - Please try to  align the elements according to the requirement, you should use Flex-box in all 2 sections(toolbar and headline section.)
- Both toolbar and headline section must have the margins (top, bottom, left, right) defined in the Template design.
-  You didn't follow the Figma design Requirement, You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)

Your project looks like this :  
awes
 The Figma design Look like this : 
Figma design

File: README.md [hunk 4]
```diff
@@ -3,7 +3,7 @@
 
 My Portfolio
 
-This project is created to show my personal projects
+This project is created to show my personal Portfolio.
```
Comment (Abenezer-Tilahun) [inline README.md:7]: - [ ] Good job for try to update professional README. But README file should be descriptive and all main headers from the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) should be present in a your repo and please try to customize.

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @lolashot   

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

✔️ No linter errors.
✔️ You used correct Github Flow.
✔️ Good commit messages

almost

## Required Changes ♻️
_Check the comments below the review._

- [x] README file should be descriptive and  all main headers from the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) should be present in your Repo. 
- [x] Pull Request should have a descriptive title telling you what changes are inside and should have a summary describing introduced changes in more detail.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MahmoudBakr23 — COMMENTED** [review]
Hi @lolashot  😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

**Highlights'**
- Well done making the requested changes from the previous reviewer. 

**Note ⚠**
- Please don't merge the PR unless it's approved! 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**